Sources/API/Routes/BalanceRoutes.swift

```diff
@@ -47,12 +49,11 @@ public final class BalanceRoutes {
      
      - returns: A StripeRequest<> item which you can then use to convert to the corresponding node
      */
-    public func history(forFilter filter: StripeFilter?=nil) throws -> StripeRequest<BalanceHistoryList> {
+    public func history(forFilter filter: StripeFilter?) throws -> StripeRequest<BalanceHistoryList> {
```

Comment:
The reason for having this `=nil` was to prevent the method from looking like `history(forFilter: nil)` if you didn't need one. In my opinion doing `history()` when you don't need to filter anything looks cleaner. Thoughts?